terraform-dev/infrastructure/variables.tf

```diff
@@ -102,7 +102,7 @@ variable "common_tags" {
 
 variable "service_names" {
   description = "List of service names to use as subdomains"
-  default     = ["startup-sample-project", "ssp"]
+  default     = ["organizations-api", "org-api"]
```

Comment:
We don't need to bother renaming ssp. It was part of the original sample and should be deleted eventually. Renaming it might just confuse people into thinking it's important.